config/db.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const connectDB = async () => {
+  try {
+    const uri = process.env.MONGO_URI || "mongodb://localhost/108_express";
```

Comment:
I have opened a mongo connection url for all of us please change it and add it to the.env file @eladaviv 
MONGO_URI = "mongodb+srv://108user:108password@cluster0.hmdji17.mongodb.net/?retryWrites=true&w=majority"